example/src/main/java/com/stripe/example/activity/PayWithGoogleActivity.kt

```diff
@@ -73,17 +73,22 @@ class PayWithGoogleActivity : AppCompatActivity() {
             .addOnCompleteListener { task ->
                 viewBinding.progressBar.visibility = View.INVISIBLE
 
-                try {
-                    if (task.isSuccessful) {
-                        showSnackbar("Google Pay is ready")
-                        viewBinding.googlePayButton.isEnabled = true
-                    } else {
-                        showSnackbar("Google Pay is unavailable")
+                runCatching {
+                    task.getResult(ApiException::class.java) == true
+                }.fold(
+                    onSuccess = { isReady ->
+                        if (isReady) {
+                            showSnackbar("Google Pay is ready")
```

Comment:
Is this string user-facing? Should it be an R.string resource? (same for the other `showSnackbar()` calls)

Reply:
While not best practice, this is our example app so shouldn't be an issue.